stacker/tests/actions/test_diff.py

```diff
@@ -137,7 +140,8 @@ def test_normalize_json(self):
             '    }\n',
             '}\n'
         ]
-        self.assertEquals(normalized_template, normalize_json(template))
+        for i, line in enumerate(normalize_json(template)):
```

Comment:
This looks like functionality change in the tests - was this required for python3?

Reply:
Sorry for not describing it clearer in the commit message. I'll explain my rationale here.

The template that we see here is generated by `normalize_json`, where it uses `json.dumps` to dump a data structure. In Python 2, the default separator of the `dumps` call is `', '`, hence the extra trailing space. In Python 3, this behavior is fixed and the json does not have that extra trailing space.

On second thought, I think the best approach here is restore the old way to run tests (`self.assertEquals(normalized_template, normalize_json(template))`), but remove all the trailing space in the template in the test fixture, meanwhile, use `separators=(',', ': ')` in the `json.dumps` call to suppress the trailing space and unify the behavior in Python 2 and Python 3.